Bloxstrap/UI/ViewModels/Bootstrapper/ByfronDialogViewModel.cs

```diff
@@ -20,7 +22,10 @@ public string VersionText
         {
```

Comment:
I think it'd be better to pass version info (ex. guid, string, etc) as a constructor argument. Far cleaner and avoids having to figure out it out here.